bdb/rep.c

```diff
@@ -5303,6 +5303,8 @@ void *watcher_thread(void *arg)
         /* sleep for somewhere between 1-2 seconds */
         poll(NULL, 0, (rand() % 1000) + 1000);
     }
+
+    bdb_thread_event(bdb_state, 0);
```

Comment:
A minor thing to consider: use `BDBTHR_EVENT_START_RDONLY` for bdb_thread_event 1 and `BDBTHR_EVENT_DONE_RDONLY` for 0?

Reply:
Yeah - for better readability - will change now.